pytorch_blade/benchmark/torch-tensorrt/README.md

```diff
@@ -0,0 +1,130 @@
+
+- crnn.yml, vgg16.yml, and yolov5.yml demonstrates a configuration with all the
+  supported backends (Torch, Torch-TensorRT, TensorRT, Torch-Blade)
+
+### Supported fields
+
+| Name | Supported Values | Description |
+| --- | --- | --- |
+| backend | all, torch, torch_tensorrt, tensorrt | Supported backends for inference. |
+| input | - | Input binding names. Expected to list shapes of each input bindings |
+| model | - | Configure the model filename and name |
+| filename | - | Model file name to load from disk. |
+| name | - | Model name |
+| runtime | - | Runtime configurations |
+| device | 0 | Target device ID to run inference. Range depends on available GPUs |
+| precision | fp32, fp16 or half, int8 | Target precision to run inference. int8 cannot be used with 'all' backend |
+| calibration_cache | - | Calibration cache file expected for torch_tensorrt runtime in int8 precision |
+
+Note:
+
+- Please note that torch runtime perf is not supported for int8 yet.
+- Torchscript module filename should end with .jit.pt otherwise it will be
+  treated as a TensorRT engine.
+
+Additional sample use case:
+
+```text
+backend:
+  - torch
+  - torch_tensorrt
+  - tensorrt
+  - torch_blade
+input:
+  input0:
+    - 3
+    - 224
+    - 224
+  num_inputs: 1
+model:
+  filename: model.plan
+  name: vgg16
+runtime:
+  device: 0
+  precision:
+    - fp32
+    - fp16
+```
+
+## Comparison (@Nvidia T4)
+
+| Model | Backend        | precision   |   Median(FPS) |   Mean(FPS) |   Median-Latency(ms) |   Mean-Latency(ms) |     99th_p |     std_dev |
+|------:|:---------------|:------------|--------------:|------------:|---------------------:|-------------------:|-----------:|------------:|
+| vgg16 | Torch          | fp32        |       117.516 |     116.689 |           0.00850948 |         0.00857261 | 0.00896431 | 0.000158592 |
+| vgg16 | Torch-Blade    | fp32        |       137.365 |     137.068 |           0.00727989 |         0.00729606 | 0.00744398 | 5.56986e-05 |
+| vgg16 | Torch-TensorRT | fp32        |       140.318 |     140.042 |           0.00712666 |         0.00716695 | 0.0073506  | 0.000575365 |
+| vgg16 | TensorRT       | fp32        |       139.23  |     138.601 |           0.00718237 |         0.00721714 | 0.00762804 | 0.000127496 |
+| vgg16 | Torch          | fp16        |       246.146 |     246.007 |           0.00406262 |         0.00406507 | 0.00413304 | 2.49436e-05 |
+| vgg16 | Torch-Blade    | fp16        |       400.434 |     401.282 |           0.00249729 |         0.0024925  | 0.00253454 | 3.45862e-05 |
+| vgg16 | Torch-TensorRT | fp16        |       381.676 |     364.289 |           0.00262002 |         0.00280076 | 0.00422715 | 0.000472002 |
+| vgg16 | TensorRT       | fp16        |       384.943 |     390.188 |           0.00259779 |         0.00256511 | 0.00267093 | 7.56818e-05 |
+
+The benchmark on vgg16 shows that TorchBlade has similar performance to
+Torch-TensorRT and TensorRT.
```

Comment:
why is there a new line?

Reply:
The linter will warn the too-long length line.